seshdash/models.py

```diff
@@ -113,6 +117,8 @@ class Alert_Rule(models.Model):
                      ('pv_production','Solar Energy Produced'),
```

Comment:
These other rules should also  be modified. i.e: pv_production should be `BoM_Data_Point#pv_production`